apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,281 @@
+                            bgColor="interaction.main-subtle.default"
+                            sx={{
+                              transition: 'opacity 300ms ease-out',
+                              opacity: 0,
+                            }}
+                            _groupHover={{
+                              opacity: 0.6,
+                            }}
+                          />
+                          <VStack
+                            sx={{
+                              transition: 'opacity 300ms ease-out',
+                              opacity: 0,
+                            }}
+                            className="hover-text"
+                            w="100%"
+                            h="100%"
+                            position="absolute"
+                            top="0"
+                            left="0"
+                            justify="center"
+                            gap="0.25rem"
+                            opacity="0"
+                            color="base.content.brand"
+                            _groupHover={{
+                              opacity: 1,
+                            }}
+                          >
+                            <Icon as={BiShow} />
+                            <Text textStyle="caption-1">Click to preview</Text>
+                          </VStack>
+                        </>
+                      )}
+                    </Box>
+                    <HStack w="100%" justifyContent="space-between">
+                      <Text
+                        className="layout-title"
+                        textAlign="left"
+                        textStyle="h6"
+                        textColor={
+                          selectedLayout.layoutTypename ===
+                          layoutEntry.layoutTypename
+                            ? 'base.content.brand'
+                            : 'base.content.strong'
+                        }
+                        mb="0.5rem"
+                        sx={{ transition: 'color 300ms ease-out' }}
+                        _groupHover={{ textColor: 'base.content.brand' }}
+                      >
+                        {layoutEntry.layoutDisplayName} layout
+                      </Text>
+                    </HStack>
+                    <Text
+                      w="100%"
+                      textAlign="left"
+                      textStyle="body-1"
+                      textColor="base.content.default"
+                    >
+                      {layoutEntry.layoutDescription}
+                    </Text>
+                  </VStack>
+                )
+              })}
+            </VStack>
+          </GridItem>
+          <GridItem
+            w="100%"
+            colSpan={3}
+            bg="grey.100"
+            p="2rem 2rem 0 2rem"
+            overflow="auto"
+          >
+            <Box
+              borderRadius="8px 8px 0px 0px"
```

Comment:
nit: are designers using radii token? might be good to start so there are less things to hardcode haha. otherwise the [default chakra radii](https://v2.chakra-ui.com/docs/styled-system/theme#border-radius) might be good enough.

```suggestion
              borderRightRadius="lg"
```

Reply:
@sehyunidaaa @samanthadotdesign